sub-select.lua

```diff
@@ -216,7 +216,7 @@ local function is_valid_audio(audio, pref)
             elseif lang == "default" then
                 if audio.default then return true end
             else
-                if audio.lang and audio.lang:find(lang) then return true end
+                if audio.lang and audio.lang:lower():find(lang:lower()) then return true end
```

Comment:
The issue with converting the `lang` strings to lowercase is that the `lang` strings are patterns, and uppercase letters can be significant in patterns. Now I don't know why anyone would use a character code in a pattern for language codes, but it might be safer to not alter the patterns for now.

Reply:
This makes sense, uppercase is useful in pattern matching. So leave it as it is and let the user use the correct case, I can update the pr to add new example matches and instructions.

Reply:
The question here is whether audio.lang and sub.lang should be lowercased?

Reply:
I agree with that, it makes things easier if the language codes are all lowercase.